gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -409,6 +409,14 @@
   public static final String DEFAULT_JOB_HISTORY_STORE_USER = "gobblin";
   public static final String JOB_HISTORY_STORE_PASSWORD_KEY = "job.history.store.password";
   public static final String DEFAULT_JOB_HISTORY_STORE_PASSWORD = "gobblin";
+  /**
+   * HDFS file properties
+   */
```

Comment:
Can we move these configs to the "Writer configuration properties" section. They can be of the from "public static final String WRITER_FILE_OWNER = WRITER_PREFIX + ".file.owner"

This way we are consistent with the rest of the configuration keys.